src/admin-views/notices/install-event-tickets.php

```diff
@@ -37,17 +33,6 @@
 		<?php echo wpautop( esc_html( $description ) ); ?>
 	</div>
 
-	<button
-		id="tribe-tickets-install-plugin"
-		<?php tribe_classes( $button_classes ); ?>
-		data-plugin-slug="<?php echo esc_attr( $plugin_slug ); ?>"
-		data-nonce="<?php echo esc_attr( $ajax_nonce ); ?>"
-		data-action="<?php echo esc_attr( $action ); ?>"
-		data-redirect-url="<?php echo esc_attr( $redirect_url ); ?>"
-		data-installing-label="<?php echo esc_attr( $installing_label ); ?>"
-		data-installed-label="<?php echo esc_attr( $installed_label ); ?>"
-		data-activating-label="<?php echo esc_attr( $activating_label ); ?>"
-		data-activated-label="<?php echo esc_attr( $activated_label ); ?>"
-	><?php echo esc_html( $button_label ); ?></button>
+	<?php	Installer::get()->render_plugin_button( 'event-tickets', $action, $button_label, $redirect_url ); ?>
```

Comment:
We could keep the `plugin_slug` as template var and use that as a parameter instead of using the string `event-tickets` so that we have it living just in one place (given that this template is only for the ET notice)